app/presenters/publishing_api/services_and_information_presenter.rb

```diff
@@ -0,0 +1,53 @@
+module PublishingApi
+  class ServicesAndInformationPresenter
+    include ApplicationHelper
+
+    attr_accessor :organisation
+    attr_accessor :update_type
+
+    def initialize(organisation, update_type: nil)
+      self.organisation = organisation
+      self.update_type = update_type || "minor"
+    end
+
+    # There is no content_id defined for this content_item because it will
+    # be set by PublishContentItem.
+    def content
+      content = BaseItemPresenter.new(
+        organisation,
+        title: "Services and information - #{organisation.name}",
+        need_ids: [],
+      ).base_attributes
+
+      content.merge!(
+        base_path: "/government/organisations/#{organisation.slug}/services-information",
+        description: nil,
+        details: {},
+        document_type: "services_and_information_page",
+        public_updated_at: organisation.updated_at,
+        rendering_app: Whitehall::RenderingApp::WHITEHALL_FRONTEND,
+        schema_name: "placeholder",
+        routes: [
+          {
+            type: "exact",
+            path: "/government/organisations/#{organisation.slug}/services-information"
+          },
+          {
+            type: "exact",
+            path: "/government/organisations/#{organisation.slug}/services-information.json"
```

Comment:
We don't need this route I think (there's nothing at https://www.gov.uk/government/organisations/hm-revenue-customs/services-information.json currently).